widget/playground/src/helpers.ts

```diff
@@ -176,31 +175,31 @@ export function insertAt(
 }
 
 export function formatConfig(config: WidgetConfig) {
```

Comment:
Please check the export code component again. it seems your changes to this function (and this file) break the code indentation.

Reply:
fixed.